src/Navbar/Navbar.js

```diff
@@ -1,14 +1,23 @@
 import './Navbar.scss'
 
-function Navbar() {
-  return (
-    <nav className="navbar navbar-expand border-bottom shadow-sm px-4 justify-content-md-start justify-content-center">
-        <a href="#" className="navbar-brand fw-bold m-0">
-          <span className="brand-accent">Food</span>
-          <span>Delivery</span>
-        </a>
-    </nav>
-  )
+function Navbar(props) {
+    function handleChangeRestaurant() {
+        props.setDisplayRestaurantsOrMenu('show-restaurants')
+    }
+
+    return (
+        <nav className="navbar border-bottom shadow-sm px-4 flex-column flex-md-row">
+            <div>
+                <a href="#" className="navbar-brand fw-bold m-0">
+                    <span className="brand-accent">Food</span>
+                    <span>Delivery</span>
+                </a>
+            </div>
+            <button className={`change-restaurant-button ${props.displayRestaurantsOrMenu}`} onClick={handleChangeRestaurant}>
+                {"<< Change Restaurant"} 
```

Comment:
why is this in brackets and quotes?

Reply:
That's because otherwise it's parsed as (invalid) HTML, also prefer this to &lt&lt